### ISSUE TITLE
Download Gages cant be done multiple times

### ISSUE THREAD

**Issue by Jdgilby** [Riverscapes/QRiS#376]
Once you have downloaded gages for an area you cannot zoom to a different extent and download gages again. This is the error.
![image](https://github.com/Riverscapes/QRiS/assets/97305413/806995b1-5178-4b78-b2b5-b47a804020c7)

There is also no way to delete the gage data that was saved to clear this out and start from scratch. 

I could see this being an issue if downloading gauges does find one, but you want to be more zoomed out and capture more.

**Comment by KellyMWhitehead** [comment]
@Jdgilby The ability to download additional gages is working (I just confirmed this), however it looks like there is an error if there are no stream gages within the map window (could this be what you are experiencing?).  I will go ahead and fix this.

**Comment by KellyMWhitehead** [comment]
Fixed the issue when there are no stream gages are found within the map extent. In any case, the download tool run smoothly:
1) New gages are downloaded and saved if they do not already exist in the project
2) Gages are found, but not saved if they already exist in the project
3) User is informed if no gages were found in the current map extent.

In addition, I have added a right click menu to delete gages in the gage list.
